configs/javascript.js

```diff
@@ -202,6 +202,22 @@ module.exports = {
 		],
 
 		'wrap-iife': [ 'error', 'any' ],
+
+		'promise/always-return': 'off',
+		'promise/avoid-new': 'off',
+		'promise/catch-or-return': 'off',
+		'promise/no-callback-in-promise': 'warn',
+		'promise/no-multiple-resolved': 'error',
+		'promise/no-native': 'off',
+		'promise/no-nesting': 'warn',
+		'promise/no-new-statics': 'error',
+		'promise/no-promise-in-callback': 'warn',
+		'promise/no-return-in-finally': 'error',
+		'promise/no-return-wrap': 'error',
+		'promise/param-names': 'error',
+		'promise/prefer-await-to-callbacks': 'off',
+		'promise/prefer-await-to-then': 'off',
```

Comment:
maybe these could be warn? but would be pretty noisy in older codebases

Reply:
`prefer-await-to-callbacks` [has no description](https://github.com/eslint-community/eslint-plugin-promise/blob/main/docs/rules/prefer-await-to-callbacks.md), tat is why I disabled that rule.

`prefer-await-to-then` - could be noisy; even in `vip-cli` we sometimes use `.then`.

If you think it makes sense to convert this to a warning, let's do that.